types/react-native-auth0/index.d.ts

```diff
@@ -74,6 +74,7 @@ export interface PasswordRealmParams {
 export interface RefreshTokenParams {
     refreshToken: string;
     scope?: string;
+    [key: string]: string;
```

Comment:
I think is better to add the `?` to make it optional

Reply:
key value string pair is optional by default no need to make it optional explicitly.